Doctrine/UserManager.php

```diff
@@ -28,30 +28,41 @@ class UserManager extends BaseUserManager
     /**
      * @var string
      */
+    private $className;
```

Comment:
the name is confusing, because the actual class name is in ``class``, not in ``className`` 😄